### PR TITLE
Moving test _helpers back into library.

### PR DESCRIPTION
**Pull request by dhermes**
This is preparation for subpackage split-up.

~~**NOTE**: Has #2393 as diffbase~~

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -13,17 +13,41 @@
 # limitations under the License.
 
 import os
-import sys
```
Comment (tseaver) [inline setup.py]: Changes to `setup.py` seem to be out-of-scope for this PR.  Stray commit?

Comment (dhermes) [inline setup.py]: @tseaver It didn't need to be, but I just did it out of habit since I ripped these out of #2370.

**Comment by dhermes** [comment]
FYI, the check is not successful because I canceled it to get more Travis throughput. I still waited for 5 hours on the jobs I actually wanted to run.

**Comment by dhermes** [comment]
And now I just rebased so hopefully Travis will go green soon.

**Comment by dhermes** [comment]
@tseaver PTAL

**Comment by tseaver** [comment]
LGTM

**Pull request merged**